### PR TITLE
[BACKPORT] init backport of changie to 1.0.latest

### PR DESCRIPTION
**Pull request by McKnight-42**
### Description

backporting changie to 1.0.latest.

### Checklist

- [x] I have signed the [CLA](https://docs.getdbt.com/docs/contributor-license-agreements)
- [x] I have run this code in development and it appears to resolve the stated issue
- [x] This PR includes tests, or tests are not required/relevant for this PR
- [x] I have run `changie new` to [create a changelog entry](https://github.com/dbt-labs/dbt-core/blob/main/CONTRIBUTING.md#Adding-CHANGELOG-Entry)

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -18,4 +18,8 @@ resolves #
 - [ ] I have signed the [CLA](https://docs.getdbt.com/docs/contributor-license-agreements)
 - [ ] I have run this code in development and it appears to resolve the stated issue
 - [ ] This PR includes tests, or tests are not required/relevant for this PR
-- [ ] I have updated the `CHANGELOG.md` and added information about my change to the "dbt-snowflake next" section.
\ No newline at end of file
+<<<<<<< HEAD
+- [ ] I have updated the `CHANGELOG.md` and added information about my change to the "dbt-snowflake next" section.
+=======
+- [ ] I have run `changie new` to [create a changelog entry](https://github.com/dbt-labs/dbt-core/blob/main/CONTRIBUTING.md#Adding-CHANGELOG-Entry)
+>>>>>>> 8c8a9c2 ([Feature] add Changie to dbt-snowflake (#212))
```
Comment (emmyoop) [inline .github/pull_request_template.md:25]: You'll need to fix this.

File: .changes/1.0.0.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+## dbt-snowflake 1.0.1-rc1 - April 19, 2022
```
Comment (emmyoop) [inline .changes/1.0.0.md:1]: I think this file isn't right.  It looks like you may have combined all the 1.0.0 changelogs with the 1.0.1 changelogs.  They should not be under the same heading.

File: .changes/0.0.0.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
+## Previous Releases
+For information on prior major and minor releases, see their changelogs:
+
```
Comment (emmyoop) [inline .changes/0.0.0.md:3]: I'm wondering if this footer for the 1.0.latest branch should actually say something like "previous versions of this package lived in `dbt-core`"?  @jtcohen6 do you have an opinion here?

**Review by emmyoop — CHANGES_REQUESTED** [review]
A few tweaks

File: .github/pull_request_template.md (hunk 1 above)
Comment (McKnight-42) [reply .github/pull_request_template.md]: removed

File: .changes/1.0.0.md (hunk 2 above)
Comment (McKnight-42) [reply .changes/1.0.0.md]: split out between 1.0.0 and 1.0.1 in files then ran new changie merge

File: .changes/0.0.0.md (hunk 3 above)
Comment (McKnight-42) [reply .changes/0.0.0.md]: tried a hand at a message

File: .changes/0.0.0.md [hunk 4]
```diff
@@ -1,3 +1,3 @@
 ## Previous Releases
-For information on prior major and minor releases, see their changelogs:
+For information on prior major and minor releases of dbt-snowflake prior to 1.0.0 please see the dbt-core repo.
```
Comment (emmyoop) [inline .changes/0.0.0.md:2]:
```suggestion
For information on prior releases of dbt-snowflake prior to 1.0.0 please see the dbt-core repo.
```

There were technically no major releases before 1.0.0.

Comment (McKnight-42) [reply .changes/0.0.0.md]: Very correct! changed wording

**Review by emmyoop — APPROVED** [review]

**Pull request merged**